src/main/java/com/sampleweb/pages/GardenAndOutdoors.java

```diff
@@ -0,0 +1,100 @@
+package com.sampleweb.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.ArrayList;
+
+public class GardenAndOutdoors extends HomePage {
+
+    public String title = "Garden & Outdoors";
+    public String pageTitle = "//h1[text()=\"Garden & Outdoors\"]";
+    public String tabHeader = "//span[@class='nav-a-content'][contains(text(),'Garden & Outdoors')]";
+    public String tabHeaderLink = "//a[@class='nav-a nav-b']";
+    public String globalStore = "//img[@class='s-ref-img-sprite']";
+    public String avgCustomerReviews = "//h4[contains(text(),'Avg. Customer Review')]";
+    public String primeCheckbox = "//input[@name='s-ref-checkbox-419158031']";
+    public String primeResultsList = "//*[@id=\"search\"]/div[1]/div[2]/div/span[3]/div[1]";
+    public String primeLabelOrMoreBuyingChoices = "//*[@aria-label='Amazon Prime' or @class='a-size-base a-color-secondary']";
+    public static String PATH = "https://www.amazon.co.uk/s/ref=nb_sb_noss?url=search-alias%3Doutdoor&field-keywords=";
+
+    public GardenAndOutdoors(RemoteWebDriver driver) {
+        super(driver);
+    }
+
+    public boolean isLoaded() {
+        return driver.findElement(By.xpath(pageTitle)).isDisplayed();
+    }
+
+    public GardenAndOutdoors setSearchCriteria(String searchText) {
+        getSearchField().sendKeys(searchText);
+        return this;
+    }
+
+    public void navigateHere() {
+        setDepartmentDropdown(title);
+        setSearchCriteria("").clickSubmitButton();
+    }
+
+    public boolean openNewLinkInNewTabAndCheckIfCorrect() {
+        String tabLink = driver.findElement(By.xpath(tabHeaderLink)).getAttribute("href");
+
+        ((JavascriptExecutor)driver).executeScript("window.open()");;
+        ArrayList<String> tabs = new ArrayList<String> (driver.getWindowHandles());
+        driver.switchTo().window(tabs.get(1)); //switches to new tab
+        driver.get(tabLink);
+
+        boolean b;
+        try {
+            if (isTitleCorrect()) {
+                b = true;
```

Comment:
Can be simplified to `b = isTitleCorrect()`.

Also change the variable name to something other than `b`